tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -22,21 +22,38 @@
 
 #include "compute_tester.hpp"
 
-template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain>
-int DFT_Test<precision, domain>::test_in_place_buffer() {
+template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain, int dimms>
+int DFT_Test<precision, domain, dimms>::test_in_place_buffer() {
     if (!init(TestType::buffer)) {
         return test_skipped;
     }
 
-    descriptor_t descriptor{ size };
+    descriptor_t descriptor = descriptor_factory<descriptor_t, dimms>::get(size);
     descriptor.set_value(oneapi::mkl::dft::config_param::PLACEMENT,
                          oneapi::mkl::dft::config_value::INPLACE);
 
+
     const size_t container_size =
         domain == oneapi::mkl::dft::domain::REAL ? conjugate_even_size : size;
+    const size_t container_size_total = 
+        container_size * static_cast<size_t>(std::pow(size, dimms-1));
+    if constexpr(domain == oneapi::mkl::dft::domain::REAL){
+        if constexpr(dimms == 2){
+            std::array<std::int64_t, dimms + 1> real_strides{0, size, 1};
+            std::array<std::int64_t, dimms + 1> complex_strides{0, size/2+1, 1};
```

Comment:
```suggestion
            std::array<std::int64_t, dimms + 1> real_strides{0, 2*(size/2 + 1), 1};
            std::array<std::int64_t, dimms + 1> complex_strides{0, (size/2 + 1), 1};
```
because testing in-place, unless I'm mistaken